include/sys/cbprintf_cxx.h

```diff
@@ -52,6 +52,7 @@ static inline int z_cbprintf_cxx_is_pchar(const volatile wchar_t *)
 template < typename T >
 static inline int z_cbprintf_cxx_is_pchar(T arg)
 {
+	ARG_UNUSED(arg);
```

Comment:
same as this, seems unrelated to the missing const (should be separate PR)?

Reply:
I think that it's fine as long as it's in separate commit.

Reply:
This is in its own commit (https://github.com/zephyrproject-rtos/zephyr/pull/39932/commits/018b440cc9e950c61634578036f46e6358227a6f)